src/etw/tracing.cpp

```diff
@@ -0,0 +1,24 @@
+#include "tracing.h"
+
+// Define the GUID to use in TraceLoggingProviderRegister
+// {85A99459-1F1D-49BD-B3DC-E5EF2AD0C2C8}
+TRACELOGGING_DEFINE_PROVIDER(g_hV8JSIRuntimeTraceLoggingProvider,
+                             "Microsoft.V8JSIRuntime",
+                             (0x85a99459, 0x1f1d, 0x49bd, 0xb3, 0xdc, 0xe5,
```

Comment:
>(0x [](start = 29, length = 3)

(nit) isn't there some simple Guid conversion to get this from string?

Reply:
I tried something similar with some macros such as __uuid, UUID etc. but i didn't have success. If you know one please share so that i can update the code.